botfront/imports/api/story/stories.methods.js

```diff
@@ -18,4 +18,14 @@ Meteor.methods({
         check(story, Object);
         return Stories.remove(story);
     },
+
+    'stories.getStories'(projectId) {
+        check(projectId, String);
+        return Stories.find({ projectId }).fetch();
+    },
 });
+
+export const getStories = async (projectId) => {
```

Comment:
why did you create this method if it does the exact same thing that the one above ??

Reply:
I wrapped the meteor call, read online its a good practice to wrap meteor calls and call the wrapper function from other places. But now I also don't see much point doing so unless we are going to call that method several times.